apprentice-action/index.js

```diff
@@ -0,0 +1,33 @@
+const core = require('@actions/core');
+const github = require('@actions/github');
+const Mocha = require('mocha')
+const fs = require('fs');
+const path = require('path');
+
+const fail = (message) => {
+    core.setFailed(message);
+    process.exit(1);
+};
+
+(async () => {
+    try { 
+        const mocha = new Mocha();
+        const testDir = 'tests/'
+
+        fs.readdirSync(testDir)
```

Comment:
we could make this slightly better by using `await` so the directory isn't read synchronously.  right now, there's no need for the `async` keyword we're using for the IIFE